server/routerlicious/packages/routerlicious/src/scriptorium/index.ts

```diff
@@ -10,23 +10,33 @@ import { deleteSummarizedOps, executeOnInterval, FluidServiceErrorCode } from "@
 import { Provider } from "nconf";
 
```

Comment:
Why do we need globalDB in scriptorium? Scriptorium only stores operations which will belong to local DB right?

@pradeepvairamani FYI

Reply:
Correct. We will continue to store the ops in local DB. There seems to be a dependency on documents collection for deleting summarized ops. Since, that will be moving to global db, scriptorium now needs to be able to access both local and global db.

Reply:
> There seems to be a dependency on documents collection for deleting summarized ops

Could you elaborate a bit more?

Reply:
We will continue to store the ops in local DB, but the scriptorium is still communicating with documents collections. For example, once we use global cosmos DB to store documents collection, the scriptorium needs to go to the global cosmos DB to execute the following code: 
https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/services-utils/src/deleteSummarizedOps.ts#:~:text=const%20uniqueDocuments%20%3D%20await,).toArray()%3B

Reply:
Thanks for the clarification